.travis.yml

```diff
@@ -37,8 +38,12 @@ jobs:
   exclude:
     - python: "pypy3.5-6.0"
       env: PIP=8.1.1
+    - python: "3.8"
```

Comment:
```suggestion
    # platform.linux_distribution() is removed in Python 3.8 (bpo-28167).
    - python: "3.8"
```
The exclude should be described.

Reply:
Also, this dimension should be excluded from AppVeyor.

Reply:
Comment out it, and add a description as well.

Reply:
Done!

Reply:
Thanks!